docsite/js/brand-colors.js

```diff
@@ -0,0 +1,8 @@
+import brandProps from '../../src/props.brand-colors.js'
+
+const brandColors = document.querySelector('#brand-colors')
+
+for (const [brand, color] of Object.entries(brandProps)) {
+  const brandTitlecase = brand.replace(/-/g, ' ').replace('brand ', '')
+  brandColors.innerHTML += `<ul class="open-colors count-em color-swatch-list"><h4>${brandTitlecase}</h4><li><button class="color-swatch" style="background-color: var(${brand})" aria-label="${brandTitlecase}"></button></li></ul>`
```

Comment:
this happens on the client and not build time yeah?
can you take the output of it and drop the markup in the build, so there's no JS required to render them?

Reply:
maybe we should switch to astro or something.

Reply:
indeed, the next version of the docs is using astro, but for now, we're hand rollin